### PR TITLE
Remove operators with existing functional tests from layer creator

### PR DESCRIPTION
**Pull request by blesniewski**
Remove ops from layer creator with existing functional tests.
Those 49 operators have been checked to not introduce any issues regarding ngraph reader functional test results.

### CODE REVIEW

**Comment by ilyachur** [comment]
Could you fix CI?

File: inference-engine/src/legacy_api/src/ie_layer_validators.cpp [hunk 1]
```diff
@@ -955,7 +955,11 @@ void DetectionOutputValidator::parseParams(CNNLayer* layer) {
 
     if (layer->CheckParamPresence("code_type")) {
         std::string _code_type = layer->GetParamAsString("code_type");
-        std::vector<std::string> code_types = {"caffe.PriorBoxParameter.CENTER_SIZE", "caffe.PriorBoxParameter.CORNER"};
+        for (auto& c: _code_type)
+        {
+            c = std::tolower(c);
```
Comment (ilyachur) [inline inference-engine/src/legacy_api/src/ie_layer_validators.cpp:960]: Why do you need convert it to lover case?

**Review by ilyachur — APPROVED** [review]
In general, LGTM

File: inference-engine/src/legacy_api/src/ie_layer_validators.cpp (hunk 1 above)
Comment (blesniewski) [reply inference-engine/src/legacy_api/src/ie_layer_validators.cpp:960]: When visitor api fetches string parameter it converts all letters to lowercase, so I needed to convert it to be able to compare them.

**Review by postrational — APPROVED** [review]

**Review by jdanieck — APPROVED** [review]

File: inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp [hunk 2]
```diff
@@ -414,6 +422,84 @@ InferenceEngine::details::CNNLayerCreator::CNNLayerCreator(const std::shared_ptr
...
+    addSpecificCreator({"DetectionOutput"},
+                       [](const std::shared_ptr<::ngraph::Node> &node,
+                          const std::map<std::string, std::string> &params) -> CNNLayerPtr {
+        LayerParams attrs = {node->get_friendly_name(), "DetectionOutput",
+                            details::convertPrecision(node->get_output_element_type(0))};
+        auto res = std::make_shared<InferenceEngine::CNNLayer>(attrs);
+        res->params = params;
+        auto parseBoolStrToIntStr = [](const std::string &param) -> const std::string {
+            if (param == "true")
+            {
```
Comment (sdurawa) [inline inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp:470]: do not introduce new indentation style.

**Review by sdurawa — APPROVED** [review]

**Pull request merged**